packages/authentication/src/decorator.ts

```diff
@@ -49,5 +49,7 @@ export function authenticate(strategyName: string, options? : Object) {
  */
 export function getAuthenticateMetadata(controllerObj: Object, methodName: string): IMetadata {
   const metadataObj: AuthenticationMetadata = Reflector.getMetadata('loopback:authenticate', controllerObj, methodName);
-  return metadataObj.getMetadata();
+  if (metadataObj)
```

Comment:
> Reflector.getMetadata('loopback:authenticate', controllerObj, methodName);

By default, Reflector is already bound to `loopback`. I think we can use `authenticate` alone. 

BTW, can we reference a constant for the key?